variants/README.md

```diff
@@ -52,6 +52,13 @@ It supports self-hosted clusters and clusters managed by [EKS](https://aws.amazo
 
 This variant is compatible with Kubernetes 1.19, 1.20, and 1.21 clusters.
 
+### aws-k8s-1.20: Kubernetes 1.20 node
```

Comment:
Should we say this is a preview and not released yet? Then change it when we release an AMI?

Reply:
I don't think preview is the right word since it's something we've tested and it's intended to work just like the other k8s variants. 

As for "not released yet", I can go ahead and add that, but I feel like people should be able to look at our releases and changelogs to know what each official release contains. This applies not just for new variants but new functionality as well (going through our README and marking every unreleased change and then un-marking them after the release seems hard to keep track of).